dj_anonymizer/django_command.py

```diff
@@ -1,20 +1,16 @@
 import time
 from django.core.management.base import BaseCommand
-from django.conf import settings
 
-from apollo import settings
 from dj_anonymizer.anonymizer import Anonymizer
 
 
-class Command(BaseCommand):
+class CommandAnonymize(BaseCommand):
     def add_arguments(self, parser):
-        parser.add_argument('--action', help='"anonymize" models or "clean" models (default: both actions will be applied)')
+        parser.add_argument('--action', help='"anonymize" models or "clean" models '
+                                             '(default: both actions will be applied)')
         parser.add_argument('-s', '--soft_mode', action='store_true')
 
     def handle(self, *args, **options):
-        print 'Anonymizer found next DB settings:'
```

Comment:
why you deleted it ?
it is related to https://blog.travis-ci.com/2018-04-03-incident-post-mortem

Reply:
The same reason. It was deleted from dj_anonymizer in apollo project before me. I brought those changes to  dj_anonymizer repo. Should I save that pice of code?